Formula/bulk_extractor.rb

```diff
@@ -19,15 +19,30 @@ class BulkExtractor < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d89721d67c9460edc181f54c0c92f3dd56dd0f70cba83356b9cc3dffecb5c995"
   end
 
-  depends_on "openssl@1.1"
+  head do
+    url "https://github.com/simsong/bulk_extractor.git", branch: "main"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "openssl@3"
 
   uses_from_macos "flex" => :build
   uses_from_macos "expat"
   uses_from_macos "ncurses"
   uses_from_macos "zlib"
 
+  # Fix --disable-rar build. Remove if upstream PR is merged.
+  # PR ref: https://github.com/simsong/bulk_extractor/pull/374
+  patch do
+    url "https://github.com/simsong/bulk_extractor/commit/1a9fde225aad0fe2ffd634bdc741b4c65586297c.patch?full_index=1"
+    sha256 "1c3cd2c87bae46d3163fe526def879d0e057fb700b3909362b8356be2ba2318e"
+  end
+
   def install
-    system "./configure", *std_configure_args, "--disable-silent-rules"
+    system "./bootstrap.sh" if build.head?
+    # Disable RAR to avoid problematic UnRAR license
+    system "./configure", *std_configure_args, "--disable-rar", "--disable-silent-rules"
```

Comment:
Looks to be using a copy of RARLAB UnRAR (https://github.com/simsong/bulk_extractor/blob/main/src/rar/license.txt). I believe Debian and Fedora consider this license non-free.

There is an open enhancement that is exploring RAR5 decoders options from libarchive and Unarchiver, which may provide a DFSG-compatible alternative: https://github.com/simsong/bulk_extractor/issues/223